package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "asab_webui_shell",
-	"version": "25.1.13",
+	"version": "25.1.14",
```

Comment:
this is quite a big change (I would say a breaking one), so make the version `25.2.1`